gulp/Gulpfile.js

```diff
@@ -0,0 +1,64 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  Declare the packages that will be required and assign each one to a variable
+//
+////////////////////////////////////////////////////////////////////////////////
+
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+
+
```

Comment:
Looking good, @AlexBrown-appnovation! One style question: do the extra linebreaks have significance?